hbase/pom.xml

```diff
@@ -75,11 +75,23 @@
       <groupId>org.apache.hbase</groupId>
       <artifactId>hbase-client</artifactId>
       <version>${hbase.hbase.version}</version>
+      <exclusions>
+        <exclusion>
+          <groupId>junit</groupId>
+          <artifactId>junit</artifactId>
+        </exclusion>
+      </exclusions>
     </dependency>
     <dependency>
       <groupId>org.apache.hbase</groupId>
       <artifactId>hbase-annotations</artifactId>
       <version>${hbase.hbase.version}</version>
+      <exclusions>
```

Comment:
This is not a test dependency, so the exclude should be unnecessary. Normal dependencies do not contain test libraries.